src/sentry/static/sentry/app/routes.jsx

```diff
@@ -325,6 +323,30 @@ const projectSettingsRoutes = (
       <IndexRedirect to="data-filters/" />
       <Route path=":filterType/" />
     </Route>
+    <Route
+      key="hooks/"
+      path="hooks/"
+      name="Service Hooks"
+      componentPromise={() =>
+        import(/*webpackChunkName: "ProjectServiceHooks"*/ './views/settings/project/projectServiceHooks')}
+      component={errorHandler(LazyLoad)}
+    />
+    <Route
```

Comment:
Breadcrumbs follow the route hierarchy, so these should be nested if you wanted the breadcrumbs to reflect that

Reply:
Yeah I figured as much -- I just didnt want to deal w/ creating even more boilerplate components just to nest them.